server/lib/models/topic.go

```diff
@@ -10,6 +18,147 @@ type Topic struct {
 }
 
 func InitTopic() {
-	create_table(Topic{}, "topics", "Id")
+	t := create_table(Topic{}, "topics", "Id")
+	t.AddIndex("locale", "INDEX", []string{"Locale"})
+}
+
+const TOPIC_FETCH_LIMIT = 50
+
+func NewTopic(dbif Dbif, a *Account, req *proto.TopicCreateRequest) (*Topic, error) {
```

Comment:
__[gometalinter]__
* [golint] exported function NewTopic should have comment or be unexported
* [gotype] undeclared name: proto